waddrmgr/scoped_manager.go

```diff
@@ -817,6 +825,180 @@ func (s *ScopedKeyManager) nextAddresses(ns walletdb.ReadWriteBucket,
 	return managedAddresses, nil
 }
 
+// extendAddresses returns the specified number of next chained address from the
+// branch indicated by the internal flag.
+//
+// This function MUST be called with the manager lock held for writes.
+func (s *ScopedKeyManager) extendAddresses(ns walletdb.ReadWriteBucket,
```

Comment:
This looks like a direct copy-past of `nextAddresses`. Can we just use that function in place instead?